paasta_tools/marathon_tools.py

```diff
@@ -879,6 +879,8 @@ def wait_for_app_to_launch_tasks(client, app_id, expected_tasks, exact_matches_o
 
 def create_complete_config(service, instance, marathon_config, soa_dir=DEFAULT_SOA_DIR):
     """Generates a complete dictionary to be POST'ed to create an app on Marathon"""
+    CONFIG_HASH_BLACKLIST = set(['instances'])
```

Comment:
Probably good to write a comment about what this is for / what kinds of things are eligible to be in it, e.g. "A set of config attributes that don't get included in the hash of the config. These should be things that Paasta+Marathon knows how to change without requiring a bounce."